.classpath

```diff
@@ -13,5 +13,6 @@
 	<classpathentry kind="lib" path="lib/basicPlayer/vorbisspi1.0.2.jar"/>
 	<classpathentry kind="lib" path="lib/basicPlayer/basicplayer3.0.jar"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/libjcsi"/>
+	<classpathentry kind="lib" path="C:/Users/Claus/Downloads/log4j-1.2.17/apache-log4j-1.2.17/log4j-1.2.17.jar"/>

```

Comment:
in case this is needed, the jar file should be added to the lib directory; we might also want to look forward to change to maven or gradle. Or may be we are just fine.